src/config/options.rs

```diff
@@ -58,10 +58,10 @@ pub enum IndentStyle {
 
 #[config_type]
 /// How to place a list-like items.
-pub enum Density {
+pub enum ItemsLayout {
```

Comment:
As much as I like this change (really), unfortunately, we cannot rename this right away, since `Density` is a part of the public API of rustfmt as a library, and we can only change it on a major version bump (cc https://github.com/rust-lang/rust/issues/54504).

Could you leave a FIXME comment about renaming it on publishing 2.0?

Reply:
woops I didn't realise that